build.cake

```diff
@@ -180,6 +172,16 @@ Task("PackBinaries")
         actions.Add(() => DoPackage("Calamari", "netcoreapp3.1", nugetVersion, rid));
     }
 
+    var dotNetCorePackSettings = GetDotNetCorePackSettings();
+
+    var commonProjects = GetFiles("./source/**/*.Common.csproj");
```

Comment:
Do we need wildcards here, isn't there really only `Calamari.Common.csproj`?

Reply:
When I went back to the way we were constructing packages before, I chose to go back to using the wildcard because that was done before too. It doesn't look to me like it's needed, but it also doesn't appear to be doing any harm.